### ISSUE TITLE
rateApp() not working, "Write Review" page in AppStore cannot be opened.

### ISSUE THREAD

**Issue by csemresari** [UrbanApps/Armchair#109]
Currently Armchair's rateApp() function is not working.

In order to open the "Write Review" page in AppStore:

**_1) Use_**
```
Armchair.opensInStoreKit(false)
```

**_2) Change Armchair's reviewURLString() like below_**

```
fileprivate func reviewURLString() -> String {

        let workingReviewURL = "https://itunes.apple.com/us/app/id\(appID)?ls=1&mt=8&action=write-review"
        return workingReviewURL
}
```

**Comment by coneybeare** [comment]
Nice find. Please submit a pull request that has this as a backwards compatible value per iOS version.

**Comment by dagerydoo** [comment]
What is the plan for updating this for iOS 11? @csemresari are you working on a fully backwards compatible change?  I noticed this [forked change](https://github.com/nafetswirth/Armchair/commit/a900845ca3ba3ef467aba4b89550205b3e83eddd) which seems good enough for most of us to proceed. I'd prefer to use a formal release from this repo, but I am concerned that it lacks support for iOS 11 a good while after the update was released...
